### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by Neustradamus** [project-jedi/jcl#143] [General]
Dear @project-jedi team, @obones, @outchy, @rrossmair (not sure), @mthoma (not sure),

It is possible to create a new release build?

The latest is very old: 2.7 - Build 5676 (2015-09-15):
- https://github.com/project-jedi/jcl/releases

Thanks in advance.

**Comment by obones** [comment]
There's already the daily zips:

https://jcl.sf.net/daily

I know this is not ideal, but from our end, a release is nothing more than renaming one of those zips, no special process, no special vetting.
